src/interpret.jl

```diff
@@ -208,7 +208,16 @@ function evaluate_call_recurse!(@nospecialize(recurse), frame::Frame, call_expr:
         err = length(fargs) > 1 ? fargs[2] : frame.framedata.last_exception[]
         throw(err)
     end
-    framecode, lenv = get_call_framecode(fargs, frame.framecode, frame.pc; enter_generated=enter_generated)
+    if fargs[1] isa Method # happens on invoke, fargs[1] is the method to call
```

Comment:
Comments below are predicated on the possibility that you *won't* reconstruct an ordinary call. (But you don't need to worry about any of this if that's what you decide to do.)